pkgs/applications/misc/hyprkeys/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, installShellFiles
+}:
+
+buildGoModule rec {
+  pname = "hyprkeys";
+  version = "1.0.3";
+
+  src = fetchFromGitHub {
+    owner = "hyprland-community";
+    repo = "Hyprkeys";
+    rev = "v${version}";
+    hash = "sha256-u2NTSth9gminIEcbxgGm/2HHyzuwf/YPNQV4VzR14Kk=";
+  };
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X main.version=v${version}"
+  ];
+
+  nativeBuildInputs = [
+    installShellFiles
+  ];
+
+  vendorHash = "sha256-JFvC9V0xS8SZSdLsOtpyTrFzXjYAOaPQaJHdcnJzK3s=";
+
+  postInstall = ''
+    installShellCompletion --cmd hyprkeys \
+      --bash <($out/bin/hyprkeys completion bash) \
+      --fish <($out/bin/hyprkeys completion fish) \
+      --zsh <($out/bin/hyprkeys completion zsh)
+  '';
+
+  meta = with lib; {
+    description = "A simple, scriptable keybind retrieval utility for Hyprland";
+    homepage = "https://github.com/hyprland-community/Hyprkeys";
+    license = licenses.gpl3;
```

Comment:
Use either gpl3Plus or gpl3Only

Reply:
I'll see what @NotAShelf wants. Guess I'll also change this to the [pkgs/by-name](https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name) format in the meantime

Reply:
It's `gpl3Only` by definition I believe. We do not have any other licenses within the repository.

Reply:
gpl3Only/Plus does not refer to the presence of other licenses, but to the presence of the "or any later version" wording.

Is Hyprkeys released under GPL version 3 or any later version (sometimes called GPLv3+) or GPL version 3 only? I can't find any license header in the repo. @NotAShelf could you clarify and/or add a note and/or license headers in the repo?

Reply:
It's GPL v3 only.


See https://github.com/hyprland-community/Hyprkeys/blob/main/LICENSE

Reply:
Thanks for confirming